<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolswidget</class>
 <widget class="QWidget" name="toolswidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 45, 54);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical{
    width: 10px;
	background-color: rgba(0, 41, 69, 0);
    margin: 0px,0px,0px,0px;
    padding-top: 0px;		/*//隐藏上下的箭头*/
    padding-bottom: 0px;
}


QScrollBar::handle:vertical{
    width: 10px;
	
	background-color: rgb(239, 239, 239);
    /*滚动条两端变成椭圆 */
    border-radius: 2px;
    min-height: 0;
}

QScrollBar::sub-page:vertical {
    
	background-color: rgba(255, 255, 255, 0);
}
QScrollBar::add-page:vertical 
{
    background-color: rgba(255, 255, 255, 0);
}

QScrollBar::add-line:vertical {
	border: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
    border: none;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::down-arrow:vertical {
    border:none;
}

QScrollBar::up-arrow:vertical {
    border:none;
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_json">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTextEdit" name="textEdit_json_source">
          <property name="font">
           <font>
            <family>OPPOSans B</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;OPPOSans B&quot;;
color: rgb(194, 194, 194);</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'OPPOSans B'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_json_target">
          <property name="font">
           <font>
            <family>OPPOSans B</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;OPPOSans B&quot;;
color: rgb(194, 194, 194);</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'OPPOSans B'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_xml">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTextEdit" name="textEdit_xml_source">
          <property name="font">
           <font>
            <family>OPPOSans</family>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_xml_target">
          <property name="font">
           <font>
            <family>OPPOSans</family>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_diff">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTextEdit" name="textEdit_diff_source">
          <property name="font">
           <font>
            <family>OPPOSans</family>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_diff_target">
          <property name="font">
           <font>
            <family>OPPOSans</family>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
